packages/guides-theme-bootstrap/resources/template/body/menu/navbar/menu-level.html.twig

```diff
@@ -3,8 +3,8 @@
     {% apply spaceless %}
         <li class="nav-item">
             <a href="{{ renderLink(entry.url) }}"
-               class="nav-link {% if entry.options.active %} active {% endif %}"
-                    {% if entry.options.active %}  aria-current="page" {% endif %} >
+               class="nav-link  {%- if menu.currentPath == entry.url %} current {%- endif -%}{% if entry.url in menu.rootlinePaths %} active {%- endif -%}"
+                    {%- if menu.currentPath == entry.url %} aria-current="page" {% endif %} >
                 {{ renderNode(entry.value.value) }}
             </a>
```

Comment:
Maybe we should extract this to a menu item link template? to remove the duplication in the templates.